Source/WTF/wtf/FunctionDispatcher.h

```diff
@@ -26,6 +26,7 @@
 #pragma once
 
 #include <wtf/Function.h>
+#include <wtf/RefCounted.h>
```

Comment:
This seems counter-intuitive.

A RefCountedSerialFunctionDispatcher will typically be inheriting from ThreadSafeRefCounted, yet we include RefCounted.h

I think it would be better to not use the macro here nor include the header and wherever ThreadSafeRefCounted would be expected to be used.

Reply:
RefCountedSerialFunctionDispatcher declares a virtual refcounting interface. It has no opinion about whether a concrete implementation is thread-safe or not.

If we really believe that all subclasses want or need thread safe refcounting, we should just declare that, and not a virtual interface.